modules/Rock.cmake

```diff
@@ -401,6 +402,14 @@ macro(rock_target_setup TARGET_NAME)
     set_property(TARGET ${TARGET_NAME}
         PROPERTY DEPS_PUBLIC_CMAKE ${${TARGET_NAME}_PUBLIC_CMAKE})
 
+    if (NOT ${TARGET_NAME}_LANG_C)
+        CHECK_CXX_COMPILER_FLAG("-Wnon-virtual-dtor" CXX_SUPPORTS_NON_VIRTUAL_DTOR)
```

Comment:
Any reason why you transformed a call to the `rock_add_compiler_flag...` helper into separate cmake calls ?